sdk/appconfiguration/Azure.Data.AppConfiguration/src/Azure.Data.AppConfiguration.csproj

```diff
@@ -4,7 +4,7 @@
     <Description>This is the Microsoft Azure Application Configuration Service client library</Description>
     <AssemblyTitle>Microsoft Azure.Data.AppConfiguration client library</AssemblyTitle>
     <Version>1.0.0-preview.5</Version>
-    <PackageTags>Microsoft Azure Application Configuration, $(PackageCommonTags)</PackageTags>
+    <PackageTags>Microsoft Azure Application Configuration;$(PackageCommonTags)</PackageTags>
```

Comment:
> Microsoft Azure Application Configuration

This doesn't work with NuGet, server still splits them: 

![image](https://user-images.githubusercontent.com/1697911/68983199-bf20cd00-07be-11ea-814e-a00856bbde49.png)

Reply:
For now I will leave as-is. We can tweak this later.